crates/build/re_types_builder/src/codegen/cpp/mod.rs

```diff
@@ -2799,3 +2838,7 @@ fn quote_deprecation_ignore_start_and_end(
 fn archetype_component_descriptor_constant_ident(obj_field: &ObjectField) -> Ident {
     format_ident!("Descriptor_{}", obj_field.name)
 }
+
+fn is_blueprint_type(obj: &Object) -> bool {
+    obj.pkg_name.contains("blueprint")
```

Comment:
FYI I think we use `obj.scope()` or something everywhere else -- not sure what the difference is.